account/src/main/resources/org/killbill/billing/account/dao/AccountEmailSqlDao.sql.stg

```diff
@@ -38,9 +38,9 @@ and t.is_active
 >>
 
 markEmailAsDeleted() ::= <<
-update <tableName()> t
-set t.is_active = 0
-where <idField("t.")> = :id
-<AND_CHECK_TENANT("t.")>
+update <tableName()>
```

Comment:
Reference: http://www.postgresql.org/docs/9.4/static/sql-update.html

Do not include the table's name in the specification of a target column — for example, UPDATE tab SET tab.col = 1 is invalid.